api/libproject/cloudsqlproxycontext.rb

```diff
@@ -0,0 +1,28 @@
+require_relative "../../libproject/utils/common"
+
+class CloudSqlProxyContext
+  attr_reader :env_file_path
+
+  def initialize(gcc)
+    @bucket_name = "#{gcc.project}-credentials"
+    @env_file_path = "db/vars.#{gcc.project}.env"
```

Comment:
Should we make this a temp file like we were before:

https://github.com/all-of-us/workbench/blob/master/api/libproject/devstart.rb#L160

so it's not in the source tree?

We also don't necessarily need the filename to vary per environment.

Reply:
That would be nice, but tricky. I'd have to copy the file, then pull it out of the container. Since we're already mounting our source tree, that's the easiest way to get files in and out. Suggestions?

Reply:
The Ruby itself isn't running inside Docker... you can just write the file to a temp file and read it like we are in read_db_vars. Right?

Reply:
The Ruby is not, but the gcloud command is. In any case, I figured out a workaround.